app/events/filters/macros/page.tsx

```diff
@@ -0,0 +1,109 @@
+'use client'
+import { useTitle } from 'react-use'
+import { Loading, LoadingFailed } from '@/common/Loader'
+import {
+  useFilterMacroList,
+  useFilterMacroRemoveMutation,
+} from '@/mod-event/useFilterMacrosList'
+import { Card } from '@/common/Card'
+import { ActionButton, LinkButton } from '@/common/buttons'
+import { PlusIcon, TrashIcon } from '@heroicons/react/24/solid'
+import { ConfirmationModal } from '@/common/modals/confirmation'
+import { useState } from 'react'
+import { EmptyDataset } from '@/common/feeds/EmptyFeed'
+
+const dateFormatter = new Intl.DateTimeFormat('en-US', {
+  dateStyle: 'medium',
+  timeStyle: 'short',
+})
+
+export default function EventFiltersMacrosListPage() {
+  const [isOpen, setIsOpen] = useState(false)
+  const { data: macroList, isFetching, error } = useFilterMacroList()
+  const {
+    mutateAsync: removeMacro,
+    error: removeMacroError,
+    isLoading: isRemovingMacro,
+  } = useFilterMacroRemoveMutation()
+
+  let pageTitle = `Moderation Filter Macros`
+  useTitle(pageTitle)
```

Comment:
Any issue with this?
```suggestion
  useTitle('Moderation Filter Macros')
```

Reply:
no issues, just built up a habbit of doing this so that if and when we do end up making the title dynamic because of whatever reason, it's 1 less line of change lo.